docs/source/extensions.rst

```diff
@@ -0,0 +1,61 @@
+- `Sharing Strategies: standard practices and strategies for sharing format extensions for NWB <https://docs.google.com/document/d/e/2PACX-1vSpLLPQV2XlfT-Qnpi_aqLPJzRjCko6Ur0U5COCEAQg5uLIN0h5vej5EPtsf6UNx1qiAIKXPiIveSWo/pub>`_
+- `Versioning Guidelines: requirements and strategy for versioning namespaces for the NWB core schema and extensions <https://docs.google.com/document/d/e/2PACX-1vSH72zNSUBToVcZDRI4gF7h15ImWRffvj-ju1oEbxggPrEFJd5L6GQc-fRiVmIi42U742tgjcRk65jv/pub>`_
+
+NWB Enhancement Proposals (NEPs)
+================================
+
+The NWB data standard is not static but evolves to adapt to changing needs from the neuroscience community, 
+clarify ambiguities, and fix bugs, among others. Users can propose smaller changes, such as improvements to
+documentation or addition of a new optional field to an existing data type, on the 
+:nwb-schema-src:`NWB Schema GitHub repository <>`. Users can propose larger changes, such as addition of 
+several new data types to support a new data modality, or a major restructuring of existing data types, 
+through NWB Enhancement Proposals (NEPs).
+
+NEPs are often collaborative and led by members of the community to address a significant need by the 
+community. Here is a list of NEPs that have been started:
+
+.. list-table:: NEPs
+   :widths: 15 45 25 15
+   :header-rows: 1
+
+   * - NEP
+     - Title
+     - Lead
+     - URLs
+   * - NEP001
+     - Events and TTL Data
+     - Ryan Ly
+     - `Google Doc <https://docs.google.com/document/d/1qcsjyFVX9oI_746RdMoDdmQPu940s0YtDjb1en1Xtdw/edit?usp=sharing>`_
+   * - NEP002
+     - Probe devices and channel mapping in extracellular ephys
+     - Alessio Buccino
+     - `Google Doc <https://docs.google.com/document/d/1q-haFEEHEgZpRoCzzQsuSWCKN4QfMsTzLnlptLaf-yw/edit?usp=sharing>`_
+
+To add a new NEP to this list, please file an issue on the :nwb-overview-src:`NWB Overview GitHub repo <>`
+or create a pull request that modifies the table.
+
+Review of NEPs for acceptance to the NWB core data standard is led by the 
+:nwb-tab:`NWB Technical Advisory Board (TAB) <>`
+and involves several phases. To submit a NEP for review and explore previous NEP reviews, see the 
+:nep-review:`NEP Review GitHub repo <>`. For more information on the NEP process, see the following links:
+
+- `Proposal Review Process: process by which extensions to the NWB core standard are proposed, evaluated, reviewed, and accepted <https://docs.google.com/document/d/e/2PACX-1vR7v4ixgnaCsJSbKji5eGWxb5muzV1M82zA-D2IswZD_KOt7HiUjcXKpTko0lqcBAD-MTd44rqFCf-V/pub>`_
+- `Working Groups for Evaluating NEPs Policy: process for evaluation and review of NWB Extension Proposals (NEPs) by a review working group (RWG) to provide guidance and a formal framework for RWG members. <https://docs.google.com/document/d/e/2PACX-1vTpDnWFpD2YDuYKXzd-6svH6ceXNBz4wOauoZivvZpQgLPYBz6yv7-eihJceBtgGTDV_TcMX9xboNsm/pub>`_
```

Comment:
Has the NEP template form been released yet? If so, please add it here. If not, it would be good to release that document

Reply:
Done